karma.config.js

```diff
@@ -1,30 +1,38 @@
+const globby = require('globby');
+const istanbul = require('rollup-plugin-istanbul');
 const { nodeResolve } = require('@rollup/plugin-node-resolve');
 
 process.env.CHROME_BIN = require('puppeteer').executablePath();
 
-let testFiles = '';
+let testFilesPattern = './test/**/*.spec.js';
+
 const matchArg = process.argv.indexOf('--match');
 if (matchArg > -1) {
-  testFiles = process.argv[matchArg + 1] || '';
+  testFilesPattern = process.argv[matchArg + 1] || '';
 }
-if (!testFiles) {
-  testFiles = 'test/**/*.spec.js';
+
+if (globby.sync(testFilesPattern).length) {
+  // eslint-disable-next-line no-console
+  console.log(`\nTesting files matching "${testFilesPattern}"\n`);
+} else {
+  // eslint-disable-next-line no-console
+  console.error(`\nNo test files matching "${testFilesPattern}"\n`);
+  process.exit(0);
```

Comment:
Exit with `0` instead of `1`?

Reply:
No problem, but this came directly from the config in \@locker/integration-karma. I'd be happy to update both.

Reply:
Ah naw. If there's precedent we'll keep that.